.php_cs.dist

```diff
@@ -6,11 +6,43 @@ $finder = PhpCsFixer\Finder::create()
 ;
 
 return PhpCsFixer\Config::create()
+    ->setRiskyAllowed(true)
     ->setRules([
         '@Symfony' => true,
+        '@Symfony:risky' => true,
+        'array_syntax' => [
+            'syntax' => 'short',
+        ],
+        'braces' => [
+            'allow_single_line_closure' => true,
+        ],
+        'declare_strict_types' => true,
+        'modernize_types_casting' => true,
+        // 'native_function_invocation' => true,
+        'no_extra_consecutive_blank_lines' => [
+            'break',
```

Comment:
We cannot enforce a blank line before/after `break;`, but we can enforce `break;` always sticking to the next `case` statement:

```php
switch ($str) {
    case 'A':
        $this->doSomething();

        $this->doSomethingElse();

        break;
    case 'B':
        $this->doSomethingAwesome();
        break;
}
```